presto-main/src/main/java/com/facebook/presto/execution/QueryManagerConfig.java

```diff
@@ -394,4 +411,10 @@ public QueryManagerConfig setRequiredWorkersMaxWait(Duration requiredWorkersMaxW
         this.requiredWorkersMaxWait = requiredWorkersMaxWait;
         return this;
     }
+
+    public enum MaterializeExchangesStrategy
```

Comment:
What about put this `MaterializeExchangesStrategy` into `FeaturesConfig.java` ? -- So it sits together with `JoinReorderingStrategy` and `JoinDistributionType` :)

Reply:
I think the idea was to keep enum in the same configuration class where it is used. All the enums are there used in the `SystemSessionProperties`. But if we want to group them by their usage in the `SystemSessionProperties` we should move them there then.